filament/backend/src/opengl/OpenGLProgram.cpp

```diff
@@ -62,6 +64,11 @@ OpenGLProgram::OpenGLProgram(OpenGLDriver& gld, Program&& program) noexcept
         lazyInitializationData->uniformBlockInfo = std::move(program.getUniformBlockBindings());
     }
 
+    // We only keep the push constants for vertex stage because we'd like to keep this class a
+    // certain size.
+    lazyInitializationData->pushConstants =
+            std::move(program.getPushConstants(ShaderStage::VERTEX));
+
```

Comment:
I don't understand this comment, this will probably break if we don't have the two sets?